draft-js-emoji-plugin/package.json

```diff
@@ -48,7 +48,6 @@
   },
   "peerDependencies": {
     "draft-js": "^0.10.1 || ^0.11.0",
-    "react": "^15.5.0 || ^16.0.0-rc",
-    "react-dom": "^15.5.0 || ^16.0.0-rc"
+    "react": "^16.3.0"
```

Comment:
react-dom is still a peer dep no?

Breaking change so major version has to increase

Reply:
react-dom is not used in this plugin. Yes I know about major version. This is not release PR yet.